mail-client/alot/alot-0.6.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+SRC_URI="https://github.com/pazz/alot/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+DEPEND="
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/mock[${PYTHON_USEDEP}] )
+	"
+RDEPEND="
+	>=app-crypt/gpgme-1.8.0-r3[python,${PYTHON_USEDEP}]
+	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	>=dev-python/twisted-core-10.2.0[${PYTHON_USEDEP}]
+	>=dev-python/urwid-1.1.0[${PYTHON_USEDEP}]
+	>=dev-python/urwidtrees-1.0[${PYTHON_USEDEP}]
+	>=net-mail/notmuch-0.13[crypt,python]
+	dev-python/python-magic[${PYTHON_USEDEP}]
+	net-mail/mailbase
+	"
+
+DOCS=( README.md )
+
+ALOT_UPDATE=""
```

Comment:
try without defining this in global scope

Reply:
Yes and the defined variable is only used once and for out-of-the-tree version of alot 0.3.2 and oldest available is 0.3.7-r1 so we might get rid of that at some point.

Reply:
Here's the diff with the last stable version in the tree
```diff
└─> diff -up alot-0.5.1.ebuild alot-0.6.ebuild                                                                                                            
                                                                                                                                                          
-                                                                                                                                                         
-src_compile() {                                                                                                                                          
-       distutils-r1_src_compile                                                                                                                          
-                                                                                                                                                         
+python_compile_all() {                                                                                                                                   
        if use doc; then                                                                                                                                  
-               pushd docs || die                                                                                                                         
-               emake html                                                                                                                                
-               popd || die                                                                                                                               
+               emake -C docs html                                                                                                                        
+               HTML_DOCS=( docs/build/html/. )                                                                                                           
        fi                                                                                                                                                
 }                                                                                                                                                        
                                                                                                                                                          
-src_install() {                                                                                                                                          
+python_src_install_all() {                                                                                                                               
        distutils-r1_src_install                                                                                                                          
                                                                                                                                                          
        dodir /usr/share/alot                                                                                                                             
        insinto /usr/share/alot                                                                                                                           
        doins -r extra                                                                                                                                    
-                                                                                                                                                         
-       if use doc; then                                                                                                                                  
-               dohtml -r docs/build/html/*                                                                                                               
-       fi                                                                                                                                                
 }                                                                                                                                                        
                                                                                                                                                          
 pkg_postinst() {                                                                                                                                         
-       if [[ ${ALOT_UPDATE} = yes ]]; then                                                                                                               
+       if has_version "<${CATEGORY}/${PN}-0.3.2"; then                                                                                                   
                ewarn "The syntax of theme-files and custom tags-sections of the config"                                                                  
                ewarn "has been changed.  You have to revise your config.  There are"                                                                     
                ewarn "converter scripts in /usr/share/alot/extra to help you out with"   
```